### ISSUE TITLE
Change Follow button to Follow Papers Button

### ISSUE THREAD

**Issue by profstein** [cuny-academic-commons/social-paper#90]
Also change Unfollow to Unfollow Paper

Due to the placement of the button in line with the author this change helps to make sure people don't think they are following the author instead of the paper.

![image](https://cloud.githubusercontent.com/assets/6502138/11383489/76b2039c-92d6-11e5-95e5-e0714969990c.png)

**Comment by boonebgorges** [comment]
Making the change it this point would affect much of the documentation that's already been prepared, so we're going to postpone this change.

**Comment by profstein** [comment]
Ok, we do Think that there is potential for confusion so if you could apply in an early dot release that would be nice.

On Nov 25, 2015, at 4:45 PM, Boone Gorges <notifications@github.com<mailto:notifications@github.com>> wrote:

Making the change it this point would affect much of the documentation that's already been prepared, so we're going to postpone this change.

##

**Comment by boonebgorges** [comment]
Just to clarify - who is "we"? Is it you and @samradd ?

It's technically easy for me to make the change, but changes like this have to be reflected across our docs, which is time-consuming. This is why we had a deadline for functional and design changes :) But yes, let's try to coordinate with the team on a release that will require docs changes - ideally, they'll be grouped together, to minimize the effort required of the documentation team.

**Comment by profstein** [comment]
Yes @samradd is the other part of we. And I'm on board with collecting these changes so they can be documented properly and more efficiently.

On Nov 25, 2015, at 5:20 PM, Boone Gorges <notifications@github.com<mailto:notifications@github.com>> wrote:

Just to clarify - who is "we"? Is it you and @samraddhttps://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_samradd&d=AwMCaQ&c=nI61yajbN8Wpmagq-MfhuPVQx6fheJzyqdJgIHizQCg&r=TYqAlEczAGrDMU8DN5yGUjYNQBifq4FtNemzKj8b9hk&m=c-v61l8Iql9MmJQEnX4yNnOvSZwoNdYvoKq3JBfqZSE&s=nQ2YavAEdd74Gg0MVvxqt6THn2_cJyIHdzxYddJ5Al0&e= ?

It's technically easy for me to make the change, but changes like this have to be reflected across our docs, which is time-consuming. This is why we had a deadline for functional and design changes :) But yes, let's try to coordinate with the team on a release that will require docs changes - ideally, they'll be grouped together, to minimize the effort required of the documentation team.

##
